src/sql/workbench/contrib/executionPlan/browser/azdataGraphView.ts

```diff
@@ -323,6 +330,31 @@ export class AzdataGraphView {
 		if (node.description) {
 			diagramNode.description = node.description;
 		}
+
+		if (node.cost) {
```

Comment:
why do we need to null check before assigning?  are we trying to preserve the values in `diagramNode` when the corresponding value is null in `node`?  If not, I think directly assigning all these properties regardless of state is a more compact/efficient/readable style.  If we are trying to preserve values (or there is some cost to assigning values, e.g. expensive property setters, etc.) then this approach makes sense to me.